### DISCUSSION TITLE
Override css styling

### DISCUSSION THREAD

**Discussion by lsherman98** [samhirtarif/react-audio-recorder#118] [General]
Is there a way to override the css styling of the AudioRecorder Component? Too make it bigger for instance etc

**Comment by noahbazer** [comment]
You can directly adjust that in the audio-recorder.css

Since the component itself consists of a container and a child icon, You can adjust the size of the icon, and the width / height of the container. These are the rules I adjusted to make the component larger:

```
.audio-recorder {
  border-radius: 40px;
  width: 80px;
}

.audio-recorder-mic {
  height: 50px;
  padding: 15px;
}

```

**Reply by noahbazer** [reply]
Of course, you can adjust these to your liking
